File: app/src/main/java/com/practice/todo/util/InMemoryCache.java
package com.practice.todo.util;

import android.location.Location;

/**
 * 内存缓存
 */
public final class InMemoryCache {

    public static final class RemindTimeCache {
        public static long remindTimeMills;
        public static int itemDbId;
    }

    public static final class RemindLocationCache {
        public static Location remindLocation;
        public static int itemDbId;
    }
}

File: app/src/main/java/com/practice/todo/util/InMemoryCache.kt
package com.practice.todo.util

import android.location.Location

/**
 * Created by suikajy on 2019.10.3
 */
object InMemoryCache {

    object RemindTimeCache {
        var remindTimeMills: Long = 0L
        var itemDbId: Int = 0
    }

    object RemindLocationCache {
        var remindLocation: Location? = null
        var itemDbId: Int = 0
    }

}